src/debug.c

```diff
@@ -1763,6 +1764,16 @@ void logStackTrace(void *eip, int uplevel) {
 
 #endif /* HAVE_BACKTRACE */
 
+sds genClusterDebugString(sds infostring){
+    infostring = sdscatprintf(infostring, "\r\n# Cluster info\r\n");
+    infostring = genClusterInfoString(infostring); 
+
+    infostring = sdscatprintf(infostring, "\r\n# Cluster Nodes\r\n");
```

Comment:
I think the cluster nodes should be formatted more similar to how the high level debug sections look. Something like:
```
------ CLUSTER NODES OUTPUT ------
```